docs/core/install/linux-package-manager-ubuntu-2004.md

```diff
@@ -0,0 +1,111 @@
+
+[!INCLUDE [package-manager-intro-sdk-vs-runtime](includes/package-manager-intro-sdk-vs-runtime.md)]
+
+## Add Microsoft repository key and feed
+
+Before installing .NET, you'll need to:
+
+- Add the Microsoft package signing key to the list of trusted keys.
+- Add the repository to the package manager.
+- Install required dependencies.
+
+This only needs to be done once per machine.
+
+Open a terminal and run the following commands.
+
+```bash
+wget https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
+sudo dpkg -i packages-microsoft-prod.deb
+```
+
+## Install the .NET Core SDK
+
+Update the products available for installation, then install the .NET Core SDK. In your terminal, run the following commands.
+
+```bash
+sudo apt-get update
+sudo apt-get install apt-transport-https
+sudo apt-get update
+sudo apt-get install dotnet-sdk-3.1
+```
+
+> [!IMPORTANT]
+> If you receive an error message similar to **Unable to locate package dotnet-sdk-3.1**, see the [Troubleshoot the package manager](#troubleshoot-the-package-manager) section.
+
+## Install the ASP.NET Core runtime
+
+Update the products available for installation, then install the ASP.NET Core runtime. In your terminal, run the following commands.
+
+```bash
+sudo apt-get update
+sudo apt-get install apt-transport-https
+sudo apt-get update
+sudo apt-get install aspnetcore-runtime-3.1
+```
+
+> [!IMPORTANT]
+> If you receive an error message similar to **Unable to locate package aspnetcore-runtime-3.1**, see the [Troubleshoot the package manager](#troubleshoot-the-package-manager) section.
+
+## Install the .NET Core runtime
+
+Update the products available for installation, then install the .NET Core runtime. In your terminal, run the following commands.
+
+```bash
+sudo apt-get update
+sudo apt-get install apt-transport-https
+sudo apt-get update
+sudo apt-get install dotnet-runtime-3.1
+```
+
+> [!IMPORTANT]
+> If you receive an error message similar to **Unable to locate package dotnet-runtime-3.1**, see the [Troubleshoot the package manager](#troubleshoot-the-package-manager) section.
+
+## How to install other versions
+
+[!INCLUDE [package-manager-switcher](./includes/package-manager-heading-hack-pkgname.md)]
+
+## Troubleshoot the package manager
+
+This section provides information on common errors you may get while using the package manager to install .NET Core.
+
+### Unable to locate
+
+If you receive an error message similar to **Unable to locate package {the .NET Core package}**, run the following commands.
+
+```bash
+sudo dpkg --purge packages-microsoft-prod && sudo dpkg -i packages-microsoft-prod.deb
+sudo apt-get update
+sudo apt-get install {the .NET Core package}
+```
+
+If that doesn't work, you can run a manual install with the following commands.
+
+```bash
+sudo apt-get install -y gpg
+wget O- https://packages.microsoft.com/keys/microsoft.asc | gpg --dearmor -o microsoft.asc.gpg
```

Comment:
Small typo: this should be `wget -O- ...` note the missing '-' before 'O'.

Reply:
Looks like we started missing it in 19.10 with this change, appears to be a typo since the earlier versions did keep the starting `-`: https://github.com/dotnet/docs/pull/17485/files#diff-5494a1067982ffbfdd34075c78e63561R98

Reply:
good catch

Reply:
Happy to help 👍